common/test/acceptance/tests/studio/test_studio_textbooks.py

```diff
@@ -43,9 +44,22 @@ def test_new_textbook_upload(self):
         And I have uploaded a PDF textbook and save the new textbook information
         Then the "View Live" link contains a link to the textbook in the LMS
         """
-        self.textbook_page.open_add_textbook_form()
-        self.textbook_page.upload_pdf_file('textbook.pdf')
-        self.textbook_page.set_input_field_value('.edit-textbook #textbook-name-input', 'book_1')
-        self.textbook_page.set_input_field_value('.edit-textbook #chapter1-name', 'chap_1')
-        self.textbook_page.click_textbook_submit_button()
-        self.assertTrue(self.textbook_page.is_view_live_link_worked())
+        self.textbook_upload_page.upload_new_textbook()
+        self.assertTrue(self.textbook_upload_page.is_view_live_link_worked())
+
+    @attr('a11y')
+    def test_textbook_page_a11y(self):
+        """
+        Uploads a new textbook, and runs an accessibility test on the pdf viewer in lms
+        """
+        self.textbook_upload_page.upload_new_textbook()
+        self.textbook_view_page = TextbookViewPage(self.browser, self.course_id)
+        self.textbook_view_page.visit()
+        self.textbook_view_page.switch_to_pdf_frame(self)
```

Comment:
I was a little confused about this part. The self I'm passing in is the test object, but the method is in a page (which has no knowledge of the test object, I think?). In textbook_view.py, I can't use the get_web_driver function without the test object.